src/app/state/i18n.service.ts

```diff
@@ -1181,12 +1180,9 @@ export class I18NService {
       fr: 'Signatures récemment utilisées',
     },
   };
-  private _locale: string = DEFAULT_LOCALE;
-  private localeSource = new BehaviorSubject<string | null>(null);
-  public currentLocale = this.localeSource.asObservable();
```

Comment:
This is now used in `toolbar.component.ts`. Either re-add it here or change the toolbar to use `this._session.getLocale()`